pom.xml

```diff
@@ -59,11 +59,11 @@
     make sure the resulting binary tarball contains only
     required jars, and that no jar has an offending license.
     -->
-    <driver.version>4.16.0</driver.version>
+    <driver.version>4.16.1-SNAPSHOT</driver.version>
     <reactive-streams.version>1.0.3</reactive-streams.version>
     <reactor.version>2020.0.19</reactor.version>
     <config.version>1.4.2</config.version>
-    <netty.version>4.1.77.Final</netty.version>
+    <netty.version>4.1.94.Final</netty.version>
```

Comment:
Netty upgrade required to match the new version in the Java driver (upgraded as of JAVA-3050).  Without this we get weird mismatch errors on the tcnative usage.